server/db/seed/populateMovieDB.js

```diff
@@ -0,0 +1,97 @@
+var knex = require('./config.js').knex;
+
+var http = require("https");
+var page = 1;
+//Populate DB with most popular movies
+while (page <= 5 ) {
+
+  var options_pop = {
+    "method": "GET",
+    "hostname": "api.themoviedb.org",
+    "port": null,
+    "path": `/3/movie/popular?page=${(page)}&language=en-US&api_key=44a8710986b6c1a6c6016e7385df058d&region=US`,

```

Comment:
Suggestion to move API key to a config file and .gitignore it. Better to have it private.